src/components/main.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
+import update from 'react-addons-update';
```

Comment:
a môžeš to použiť aj na "mojích" miestach, lebo o tejto šikovnej utilitke som nevedel :-)